app/controllers/application_controller.rb

```diff
@@ -1,10 +1,18 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
-  helper_method :current_user
+  helper_method :current_user, :bounce_guest
```

Comment:
Meh.  This is a Bundayism.  Let's drop this.  Pick a better name _and_ we could use a before action for it.